secator/cli.py

```
@@ -1070,6 +1074,82 @@ def report_list(ctx, workspace, runner_type, time_delta):
+
+	# Display errors as Error output types
+	if errors_raw:
+		errors_to_show = errors_raw if show_all else errors_raw[-1:]
+		console.print()
+		extra = ', showing last 1' if not show_all and len(errors_raw) > 1 else ''
+		console.print(f'[bold]Errors[/] ({len(errors_raw)} total{extra}):')
+		for err_data in errors_to_show:
+			if isinstance(err_data, dict):
+				try:
+					err = Error.load(err_data)
+				except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)